site/_posts/2020-06-29-django-and-oauth.md

```diff
@@ -13,43 +13,43 @@ In this tutorial, we'll build a basic Django web application which does user reg
 
```

Comment:
maybe change the description to, "In this tutorial, we'll build a basic Django web application using FusionAuth for an easier and safer way at handling user registration and authentication."